### PR TITLE
Update pt-BR.txt

### PR DESCRIPTION
**Pull request by renansimoes**
If you are applying translations for an issue (or multiple), please list them below (if not, delete this text).

Applying for issue(s):
- #1400
- #1411
- #1419
- #1424

### CODE REVIEW

File: data/language/pt-BR.txt [hunk 1]
```diff
@@ -4987,4 +5134,235 @@ STR_CPTY    :4 passageiros por carro
...
+[HOVRBORD]
+STR_NAME    :Hoverboards
+
+#TT
+[HOVERBKE]
+STR_NAME    :Motos Flutuantes
+
+#TT, Reverser RC
+[POLICECR]
+STR_NAME    :Police Cars
```
Comment (Gymnasiast) [inline data/language/pt-BR.txt]: Forgot to translate.

File: data/language/pt-BR.txt [hunk 2]
```diff
@@ -58,13 +58,13 @@ STR_0053    :Montanha-Russa Tornado de Aço
 STR_0054    :Montanha-Russa de Madeira
 STR_0055    :Montanha-Russa de Fricção Lateral
 STR_0056    :Rato Maluco
-STR_0057    :Montanha-Russa Multidimensional
```
Comment (Gymnasiast) [inline data/language/pt-BR.txt]: This string, and several others, were corrected by another contributor. Could you change them back?

**Comment by renansimoes** [comment]
@Gymnasiast do you know what user is also translating to ptbr? Maybe in the next translations I can get in touch with him, so I can know when he translated something.

**Comment by Gymnasiast** [comment]
Yes, the user is @byuufx.

File: data/language/pt-BR.txt [hunk 3]
```diff
@@ -1,4 +1,3 @@
-# STR_XXXX part is read and XXXX becomes the string id number.
```
Comment (Gymnasiast) [inline data/language/pt-BR.txt]: This change is not needed.

File: data/language/pt-BR.txt [hunk 4]
```diff
@@ -526,10 +525,10 @@ STR_0521    :Uma montanha-russa rápida e retorcida com curvas fechadas e quedas
 STR_0522    :Uma montanha-russa menor onde passageiros sentam acima do trilho com nenhum carro envolta
 STR_0523    :Passageiros viajam devagar em veículos motorizados por uma rota baseada na pista
 STR_0524    :O carro de queda livre é pneumaticamente lançado para cima em uma alta torre de aço e então desce em queda livre
-STR_0525    :Passageiros descem por um trilho retorcido guiados apenas pela curvatura e inclinação do trilho semicircular
+STR_0525    :Passageiros descem por um trilho retorcido guiados apenas pela curvatura e inclinação do trilho semicircular
```
Comment (Gymnasiast) [inline data/language/pt-BR.txt]: There is still a space at the end.

File: data/language/pt-BR.txt [hunk 5]
```diff
@@ -526,10 +525,10 @@ STR_0521    :Uma montanha-russa rápida e retorcida com curvas fechadas e quedas
 STR_0522    :Uma montanha-russa menor onde passageiros sentam acima do trilho com nenhum carro envolta
 STR_0523    :Passageiros viajam devagar em veículos motorizados por uma rota baseada na pista
 STR_0524    :O carro de queda livre é pneumaticamente lançado para cima em uma alta torre de aço e então desce em queda livre
-STR_0525    :Passageiros descem por um trilho retorcido guiados apenas pela curvatura e inclinação do trilho semicircular
+STR_0525    :Passageiros descem por um trilho retorcido guiados apenas pela curvatura e inclinação do trilho semicircular 
 STR_0526    :Passageiros viajam em uma cabine de observação rotativa que viaja até o topo de uma torre alta
 STR_0527    :Uma montanha-russa suave de aço capaz de fazer loops verticais
-STR_0528    :Passageiros viajam em barcos infláveis em tubo encurvado semicircular ou completamente fechado
+STR_0528    :Passageiros viajam em barcos infláveis em tubo encurvado semicircular ou completamente fechado.
```
Comment (Gymnasiast) [inline data/language/pt-BR.txt]: None of the string in this block (512-602) have dots. You'll need to remove the dot here, or put a dot after all the other sentences in 512-602.

File: data/language/pt-BR.txt [hunk 6]
```diff
@@ -4987,4 +5133,618 @@ STR_CPTY    :4 passageiros por carro
 [CONGAEEL]
 STR_NAME    :Montanha-Russa Enguia Conger
 STR_DESC    :Uma montanha russa de aço compacta onde trens com formato de enguia viajam por parafusos e loops
-STR_CPTY    :4 passageiros por carro
```
Comment (Gymnasiast) [inline data/language/pt-BR.txt]: Please restore this string.

File: data/language/pt-BR.txt [hunk 7]
```diff
@@ -4987,4 +5133,618 @@ STR_CPTY    :4 passageiros por carro
...
+STR_NAME    :Carros Avião a Jato
+#TT
+[HOTRODXX]
+STR_NAME    :Carros Hot Rod
+
+###########
+# Scenery #
+###########
+
+#Bulrushes was spelt incorrectly
```
Comment (Gymnasiast) [inline data/language/pt-BR.txt]: Remove this entry

File: data/language/pt-BR.txt [hunk 8]
```diff
@@ -4987,4 +5133,618 @@ STR_CPTY    :4 passageiros por carro
...
+[TBR]
+STR_NAME    :Bulrushes
+
+## Start OpenRCT2 Official
+[XXBBBR01]
+STR_NAME    :Base Block
```
Comment (Gymnasiast) [inline data/language/pt-BR.txt]: Please translate these entries

File: data/language/pt-BR.txt [hunk 9]
```diff
@@ -4987,4 +5133,618 @@ STR_CPTY    :4 passageiros por carro
...
+STR_NAME    :Wooden Post Wall
+
+[ACWWF32]
+STR_NAME    :Wooden Post Wall
+
+## End OpenRCT2 Official
+
+###############################################################################
+## RCT2 Scenarios
+###############################################################################
```
Comment (Gymnasiast) [inline data/language/pt-BR.txt]: Either translate or remove these scenario entries.

**Pull request merged**